deps/rabbit/src/rabbit_mirror_queue_misc.erl

```diff
@@ -1010,7 +1010,9 @@ has_ha_policies(Policies) ->
               does_policy_configure_cmq(KeyList)
       end, Policies).
 
-does_policy_configure_cmq(KeyList) ->
+does_policy_configure_cmq(Map) when is_map(Map) ->
+    lists:keymember(<<"ha-mode">>, 1, rabbit_data_coercion:to_proplist(Map));
```

Comment:
```suggestion
    is_map_key(<<"ha-mode">>, Map);
```

Reply:
`is_map_key` ... I've never used that one 👍